infra/stack.json

```diff
@@ -0,0 +1,415 @@
+{
+    "AWSTemplateFormatVersion": "2010-09-09",
+    "Description": "",
+    "Parameters": {
+        "DBUsername": {
+            "NoEcho": "true",
+            "Description": "Username for database access",
+            "Type": "String",
+            "MinLength": "1",
+            "MaxLength": "16",
+            "AllowedPattern": "[a-zA-Z][a-zA-Z0-9]*",
+            "ConstraintDescription": "must begin with a letter and contain only alphanumeric characters."
+        },
+        "DBPassword": {
+            "NoEcho": "true",
+            "Description": "Password database access",
+            "Type": "String",
+            "MinLength": "8",
```

Comment:
**Suggestion** (non-blocking): I am curious if we should set min password length to be longer.

Reply:
I'm not too worried about it - they're randomly generated, and long.  I'll make it min 16.